2.10.x/alpine3.19/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM alpine:3.18
+FROM alpine:3.19
 
 ENV NATS_SERVER 2.10.7
```

Comment:
Should this be 2.10.8?

Reply:
yes I will update the version in this PR again once the binary is out